apps/prairielearn/src/lib/question-testing.ts

```diff
@@ -243,6 +243,7 @@ async function testVariant(
     user_id,
     authn_user_id,
     true,
+    true,
```

Comment:
Not in scope for this PR, but we should really make this an option object 😭

Reply:
Agreed this is out of scope; going to resolve this thread.